scripts/Microsoft.DotNet.Cli.Build.Framework/project.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "NETStandard.Library": "1.0.0-rc2-23805",
+        "System.Diagnostics.Process": "4.1.0-rc2-23805"
```

Comment:
Yes, @KrzysztofCwalina is going to replace it with a new Process API but didn't want the current windows-centric API to be forever legacy on new platforms.  @KrzysztofCwalina do you have an issue tracking the new API?